src/quadrature_tables/gaussquad_tri_table.jl

```diff
@@ -0,0 +1,79 @@
+# Dunavant, D. A. (1985), High degree efficient symmetrical Gaussian quadrature
+# rules for the triangle. Int. J. Numer. Meth. Engng., 21: 1129–1148. doi:
+# 10.1002/nme.1620210612
+function _get_gauss_tridata(n::Int)
+    if (n == 1)
+    xw=[0.33333333333333 0.33333333333333 1.00000000000000 / 2.0];
```

Comment:
Actually I noticed this before as well, and I think I checked that we got the correct result, did we divide by 2 somewhere else before? Instead of in this table I mean.